src/inputs/actions.js

```diff
@@ -1,6 +1,7 @@
 import OutriggerAPI from '../utils/api';
 import _ from 'underscore';
 
+export const CLEAR_INPUTS = 'CLEAR_INPUTS';
```

Comment:
I'll do remove_inputs. I think I'd like to keep it as a separate event because I think the same event should also remove the bundle.